tchannel/container/heap.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
No license header. Run scripts/install-hooks.sh in the root directory of the project to get the pre-commit hook that adds the license header.